src/shared/Button/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from './Button.module.scss'
+import { IButton } from '@shared/Button/Button.interface'
+import classNames from 'classnames'
+
+const Button: React.FC<IButton> = ({
+  url,
+  onClick,
+  disabled,
+  styleBtn,
+  children,
+}) => {
+  const classes = classNames(styles.nextBtn, {
+    [styles.nextBtnRight]: styleBtn === 'nextBtnRight',
```

Comment:
Я прихожу к тебе и спрашиваю про дизайн типы кнопок. И ты мне говоришь есть тип `nextBtnRight`, вместо того чтобы сказать primary (основная), secondary (вторичная). Название nextBtnRight вооооооооообще мне ничего не говорит в контексте типов дизайна кнопки. 

```suggestion
    [styles.primary]: designType === 'primary',
```